apps/angular/src/core/services/app-config.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { environment } from '@js-camp/angular/environments/environment';
+
+/** Works with environment variables. */
+@Injectable({ providedIn: 'root' })
+export class AppConfigService {
+	private readonly variables = environment;
+
+	/** Get baseApiUrl environment variable. */
+	public get baseApiUrl(): string {
+		return this.variables.baseApiUrl;
+	}
```

Comment:
It seems like there are no benefits in using getter here. Let's rework it to make code simpler:
```suggestion
	/** 'baseApiUrl' environment variable. */
	public readonly baseApiUrl = environment.baseApiUrl;
```